modules/tile-converter/docs/api-reference/build-instructions.md

```diff
@@ -0,0 +1,68 @@
+## npx command usage
```

Comment:
Please start with `# Build Instructions` - single hash this is picked up by the table of contents in the website.

Reply:
Can we add a header explaining why and when a user would want to build things themselves?

My thinking is that these instructions are only for advanced users? If so we want to make that clear.